src/modules/mc_hover_thrust_estimator/MulticopterHoverThrustEstimator.cpp

```diff
@@ -167,34 +182,26 @@ void MulticopterHoverThrustEstimator::Run()
 			reset();
 		}
 
-		ZeroOrderHoverThrustEkf::status status;
-		status.hover_thrust = _hover_thrust_ekf.getHoverThrustEstimate();
-		status.hover_thrust_var = _hover_thrust_ekf.getHoverThrustEstimateVar();
-		status.accel_noise_var = _hover_thrust_ekf.getAccelNoiseVar();
-		status.innov = NAN;
-		status.innov_var = NAN;
-		status.innov_test_ratio = NAN;
-
-		publishStatus(status);
+		if (_was_valid) {
+			// only publish a single message to invalidate
+			hover_thrust_estimate_s status_msg;
```

Comment:
This could also be in a function called something like `publishInvalidStatus()`

Reply:
Done.